app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
```

Comment:
In general, a `module` is the right place for this, mixed in as you guys have done - but `app/helpers` are a special case of that. You probably don't want to do that here.

Helpers are modules for storing presentation or view logic that are automatically mixed in to view files. So this `ApplicationHelper` will be mixed into every single view we create. In this particular app that adds up to zero... But generally speaking, because of that behavior, that makes this not a great place for this.

Just a plain old ruby module outside the context of helpers probably makes the most sense. I could go in `lib` or `jobs` or perhaps even`models`.

I might recommend making a new model (not `ActiveRecord` based, just a PORO) that either has this as a class method or takes in these three params in the constructor and provides a `formatted_time` method of some sort.